bluemira/equilibria/coils/_grouping.py

```diff
@@ -998,3 +1001,26 @@ def _sum(
         ind = self._control_ind if control else slice(None)
 
         return np.sum(output[..., ind], axis=-1) if sum_coils else output[..., ind]
+
+    def get_coiltype(self, ctype):
+        """Get coils by coils type"""
+        if coiltype := self._get_coiltype(ctype):
+            return CoilSet(*coiltype)
+        return None
+
+    @classmethod
+    def from_group_vecs(cls, eqdsk: EQDSKInterface):
+        """Same as its parent class,
+        initialize an instance of CoilSet from group vectors,
+        but it also automatically decide which coils are control coils
+        according to CoilType.get_actively_controlled_types.
+        """
```

Comment:
This is too verbose. See suggestion, remember this should be able to be read in isolation.
```suggestion
        """Create CoilSet from eqdsk group vectors.
        
        Automatically sets all coils that are not implicitly passive to control coils
        """
```